.ruby-version

```diff
@@ -1 +1 @@
-2.3.0
+2.3.4
```

Comment:
Question: would this impact the gem compilation or just the development tooling?